cmd/vulcanizer/shards.go

```diff
@@ -8,9 +8,14 @@ import (
 )
 
 var nodesToCheck []string
+var activeOnly bool
 
 func init() {
 	cmdShards.Flags().StringSliceVarP(&nodesToCheck, "nodes", "n", []string{}, "Elasticsearch node(s) to get shard information from")
+	cmdShardsRecovery.Flags().StringSliceVarP(&nodesToCheck, "nodes", "n", []string{}, "Elasticsearch node(s) to get shard information from")
+	cmdShardsRecovery.Flags().BoolVar(&activeOnly, "active", false, "Only display active recoveries")
```

Comment:
I'd flip this, make `active_only` true by default. IMO, the recovery output isn't that useful for the already started shards.

Reply:
🤦‍♂️ My bad on missing the unit test.  At this point I'm not sure there's a solid way of testing recovery functionality in the CI environment, but I did add a unit test. 

I've also set the default behavior for the CLI to only return the active recoveries.

Reply:
No worries, I'm not anxious about setting up all the possible scenarios in a CI env, but I like the idea of isolating as much logic as possible into methods that can just be unit tested 👍